user-guide/howto/pxe-zone.rst

```diff
@@ -0,0 +1,147 @@
+                    autoindex on;
+                }
+
+    - Create the document root directory.
+
+        .. code-block:: bash
+
+            [root@pxe-boot ~] mkdir /data/tftpboot/install
+
+* Enable both services.
+
+    .. code-block:: bash
+
+        [root@pxe-boot ~] svcadm enable dnsmasq
+        [root@pxe-boot ~] svcadm enable nginx
+
+
+Preparing TFTP and iPXE Boot Files
+##################################
+
+* The following files iPXE files should be placed into the TFTP root directory:
+
+    - iPXE client: :download:`undionly.kpxe <files/pxe/IPXE-100612_undionly.kpxe>`
+    - iPXE scripts: :download:`menu.ipxe <files/pxe/menu.ipxe>` and/or :download:`esdc-latest.ipxe<files/pxe/esdc-latest.ipxe>`
+
+    .. code-block:: bash
+
+        [root@pxe-boot ~] cd /data/tftpboot
+        [root@pxe-boot tftpboot] curl -o undionly.kpxe https://docs.danubecloud.org/user-guide/_downloads/IPXE-100612_undionly.kpxe
+        [root@pxe-boot tftpboot] curl -O https://docs.danubecloud.org/user-guide/_downloads/menu.ipxe
+        [root@pxe-boot tftpboot] curl -O https://docs.danubecloud.org/user-guide/_downloads/esdc-latest.ipxe
+
+* Download and unpack a *Danube Cloud* ErigonOS (SmartOS) platform archive.
+
+    .. code-block:: bash
+
+        [root@pxe-boot ~] mkdir /data/tftpboot/erigonos
+        [root@pxe-boot ~] cd /data/tftpboot/erigonos
+        [root@pxe-boot erigonos] curl -O https://download.erigones.org/esdc/factory/platform/platform-<version>.tgz
+        [root@pxe-boot erigonos] gtar -xzvf platform-<version>.tgz
+        [root@pxe-boot erigonos] mv platform-<version> <version>
+        [root@pxe-boot erigonos] cd <version>
+        [root@pxe-boot <version>] mkdir platform
+        [root@pxe-boot <version>] mv i86pc platform
+
+    - After this operation the kernel should be in ``/data/tftpboot/erigonos/<version>/platform/i86pc/kernel/amd64/unix``
+    - and the boot archive should be in ``/data/tftpboot/erigonos/<version>/platform/i86pc/amd64/boot_archive``.
+
+* Configure the iPXE script. The default is to use the ``menu.ipxe``, but you can also boot a compute node directly by using the example ``esdc-latest.ipxe`` script. In any case, the ``platform-version`` and ``install-host`` variables at the beginning of the *.ipxe* script should be adjusted to your reality:
+
+    .. code-block:: bash
+
+        [root@pxe-boot tftpboot] vim menu.ipxe
+
+            set platform-version <platform-version>
+            set install-host <pxe-boot-host-IP-address>
+
+
+Preparing HTTP Install Files
+############################
+
+* Download and unpack a *Danube Cloud* compute node :ref:`USB image <cn_image>`.
+
+    .. code-block:: bash
+
+        [root@pxe-boot ~] cd /data/tftpboot/install
+        [root@pxe-boot install] curl -O https://download.erigones.org/esdc/usb/stable/esdc-ce-cn-<version>.img.gz 
+        [root@pxe-boot install] gzip -d esdc-ce-cn-<version>.img.gz 
+
+* Download a sample install script.
```

Comment:
In my opinion, the install script should not be a sample. It should be a stable code.